samples/typescript_nodejs/16.proactive-messages/package.json

```diff
@@ -18,7 +18,7 @@
         "url": "https://github.com"
     },
     "dependencies": {
-        "botbuilder": "~4.14.0",
+        "botbuilder": "~4.15.0-rc0",
```

Comment:
>4.15.0-rc0" [](http://example.com/codeflow?start=24&length=11)

4.15.0 has been published.

Reply:
Hi @mrivera-ms, we updated the packages. Thanks!